utils/build-script

```diff
@@ -655,9 +655,12 @@ class BuildScriptInvocation(object):
             ]
 
             # Isolate build from the system; Darwin toolchains build against SDKs.
-            args.extra_cmake_options.append(
-                '-DCMAKE_IGNORE_PATH=/usr/lib;/usr/local/lib;/lib'
-            )
+            # For additional isolation, disable pkg-config. Homebrew's pkg-config
+            # prioritizes CommandLineTools paths, resulting in compile errors.
+            args.extra_cmake_options += [
+                '-DCMAKE_IGNORE_PATH=/usr/lib;/usr/local/lib;/lib',
```

Comment:
Is there a cmake option we can use that just does this without us needing to hack around it?

@compnerd

Reply:
I spoke with @gottesmm, he's referring to the line below:

```python
'-DPKG_CONFIG_EXECUTABLE=/usr/bin/true',
```

Reply:
This was discussed out of band, and the conclusion was that disabling `pkg-config` via `PKG_CONFIG_EXECUTABLE=/usr/bin/false` is the most direct & concise way to do this. A larger scoped solution is to use checked in CMake cache files to explicitly declare SDK dependencies, which may happen in the future.